beacon-chain/core/helpers/validators.go

```diff
@@ -152,7 +152,7 @@ func ActiveValidatorIndices(ctx context.Context, s state.ReadOnlyBeaconState, ep
 	}
 
 	if err := UpdateCommitteeCache(ctx, s, epoch); err != nil {
-		return nil, errors.Wrap(err, "could not update committee cache")
+		log.WithError(err).Error("Could not update committee cache")
```

Comment:
why are we not returning an error but we are still returning an error on https://github.com/offchainlabs/prysm/blob/92bd211e4dfb583d7d3602e21a48e38f753155d9/beacon-chain/core/helpers/validators.go#L203